fsm.h

```diff
@@ -49,20 +49,37 @@ namespace fsm
      */
     bool updateTimersFromFlash();
 
+    /**
+     * adds to the fsm queue the next dataset for the FSM to process
+     */
+    void addToSensorDataQueue(); 
+
+    /**
+     * check if the fsm queue is empty
+     * @return true if empty, false otherwise
+     */
+    bool stateQueueEmpty(); 
```

Comment:
Are both of these functions talking about the same queue? If so, what do you think about making the names more consistent?
`addToSensorDataQueue` and `isSensorDataQueueEmpty`
or 
`addToStateQueue` and `isStateQueueEmpty`